File: CS/14_Page/SplitFileByParticularPage.cs
using Spire.Pdf;
using Spire.Pdf.Actions;
using Spire.Pdf.General;
using Spire.Pdf.Graphics;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace SplitFileByParticularPage
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a pdf document
            PdfDocument oldPdf = new PdfDocument();

            //Load an existing pdf from disk
            oldPdf.LoadFromFile(@"..\..\..\..\..\..\Data\Sample.pdf");

            //Create a new PDF document
            PdfDocument newPdf = new PdfDocument();

            //Initialize a new instance of PdfPageBase class
            PdfPageBase page;

            //Specify the pages which you want them to be split
            for (int i = 1; i < 3; i++)
            {
                //Add same size page for newPdf
                page = newPdf.Pages.Add(oldPdf.Pages[i].Size, new Spire.Pdf.Graphics.PdfMargins(0));

                //Create template of the oldPdf page and draw into newPdf page
                oldPdf.Pages[i].CreateTemplate().Draw(page, new System.Drawing.PointF(0, 0));
            }

            String result = "SplitFileByParticularPage_out.pdf";

            //Save the document
            newPdf.SaveToFile(result);
            //Launch the Pdf file
            PDFDocumentViewer(result);
        }

        private void PDFDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/14_Page/SplitFileByParticularPage.vb
Imports Spire.Pdf
Imports Spire.Pdf.Graphics

Namespace SplitFileByParticularPage
    Partial Public Class Form1
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            ' Create a new instance of PdfDocument object for the original PDF
            Dim oldPdf As New PdfDocument()

            ' Load the original PDF file from the specified path
            oldPdf.LoadFromFile("..\..\..\..\..\..\Data\Sample.pdf")

            ' Create a new instance of PdfDocument object for the modified PDF
            Dim newPdf As New PdfDocument()

            ' Declare a variable to store the PdfPageBase object
            Dim page As PdfPageBase

            ' Iterate through pages 1 and 2 of the original PDF
            For i As Integer = 1 To 2
                ' Add a new page to the modified PDF with the same size as the current page in the original PDF, and set margins to 0
                page = newPdf.Pages.Add(oldPdf.Pages(i).Size, New PdfMargins(0))

                ' Draw the template of the current page in the original PDF onto the new page at coordinates (0, 0)
                oldPdf.Pages(i).CreateTemplate().Draw(page, New PointF(0, 0))
            Next i

            ' Specify the filename for the output PDF
            Dim result As String = "SplitFileByParticularPage_out.pdf"

            ' Save the modified PDF document to the output file
            newPdf.SaveToFile(result)

            ' Close the modified PDF document
            newPdf.Close()

            ' Close the original PDF document
            oldPdf.Close()

            ' Launch the Pdf file
            PDFDocumentViewer(result)
        End Sub

        Private Sub PDFDocumentViewer(ByVal fileName As String)
            Try
                Process.Start(fileName)
            Catch
            End Try
        End Sub
    End Class
End Namespace
